conf/nutch-default.xml

```diff
@@ -1207,6 +1207,15 @@
   </description>
 </property>
 
+<property>
+  <name>moreIndexingFilter.mapMimeTypes.field</name>
+  <value></value>
+  <description>It's used if moreIndexingFilter.mapMimeTypes is true. Indicates the field
+  where the mapped MIME-type must be written. If it's null, the field "type"
```

Comment:
I would prefer: `If it's empty or unset, the content of the field "type" ..."`

Of course, `conf.get("moreIndexingFilter.mapMimeTypes.field")` returns null in both cases (empty or unset). But that's only known to programmers familiar with the internals of a Hadoop configuration. Just to prevent that anybody writes literally "null" as value.